frontend/src/containers/App.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
-
 import '../styles/App.scss';
+import LandingPage from './LandingPage';
+import UniversalLandingPage from "../containers/UniversalLandingPage.js";
+import { Route, Switch } from "react-router-dom";
 
 function App() {
   return (
-    <div className='App glow'>
-      
+    <div className="App glow">
+      <Switch>
+        <Route path="/player" component={LandingPage} />
```

Comment:
Change to PlayerLandingPage instead of LandingPage to avoid confusion with HostLandingPage